Mage.Sets/src/mage/sets/AsiaPacificLandProgram.java

```diff
@@ -18,8 +18,10 @@ public static AsiaPacificLandProgram getInstance() {
     }
 
     private AsiaPacificLandProgram() {
-        super("Asia Pacific Land Program", "APAC", ExpansionSet.buildDate(1997, 10, 13), SetType.PROMOTIONAL);
+        super("Asia Pacific Land Program", "PALP", ExpansionSet.buildDate(1997, 10, 13), SetType.PROMOTIONAL);
```

Comment:
If you change set code then all set code usage must be changed too (some download or verify source code uses set code correction for third party services):

![shot_200116_223656](https://user-images.githubusercontent.com/8344157/72552674-d5c3cd00-38b0-11ea-901e-6478f4ea15cd.png)

Reply:
Thanks for notifying. Should I revert it or fix the other occurences?

Reply:
Yes, you can replace it in other places too.

Reply:
Replaced the instances. Updated the comment (first hit in your screenshot). Should be solved.